devops/helm/_clamav/values.yaml

```diff
@@ -0,0 +1,91 @@
+replicaCount: 1
+
+image:
+  repository: ghcr.io/bcgov/clamav-unprivileged
+  tag: ca3d42f3dde3c5aa9bcab636f752119bbe6a67e8 # pragma: allowlist secret
+  pullPolicy: IfNotPresent
+
+priorityClassName: ""
+
+imagePullSecrets: []
+
+nameOverride: clamav
+fullnameOverride: clamav
+
+podSecurityContext:
+securityContext:
+
+service:
+  type: ClusterIP
+  port: 3310
+
+# Ensure to have the minimum resources to run clamd
+resources:
+  limits:
+    cpu: 1000m
+    memory: 3Gi
+  requests:
+    cpu: 300m
+    memory: 2Gi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+affinityTemplate: |
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+    - weight: 100
+      podAffinityTerm:
+        topologyKey: "kubernetes.io/hostname"
+        labelSelector:
+          matchLabels: {{ include "clamav.selectorLabels" . | nindent 10 }}
+
+startupProbe:
+  initialDelaySeconds: 60
+  periodSeconds: 30
+  failureThreshold: 3
+  timeoutSeconds: 600
+
+livenessProbe:
+  initialDelaySeconds: 300
+  periodSeconds: 10
+  failureThreshold: 3
+  timeoutSeconds: 600
+
+readinessProbe:
+  initialDelaySeconds: 90
+  periodSeconds: 10
+  failureThreshold: 3
+  timeoutSeconds: 600
+
+hpa:
+  enabled: false
+  maxReplicas: 5
+  # average CPU usage utilization percentage per pod (1-100)
+  cpu: 80
+  # average Memory usage utilization percentage per pod (1-100)
+  # memory: 80
+  # average http_requests utilization per pod (value as a string)
+  # requests: 1k
+
+podDisruptionBudget:
+  enabled: false
+  # minAvailable: 1
+  # maxUnavailable: 1
```

Comment:
Maybe not part of this PR, for clarification, `hpa` as `enabled: false` and `podDisruptionBudget` as `enabled: false` will require further investigations can we have them just enabled?

Reply:
yes we have to enable them, I will make changes in the upcoming PR